src/main/java/org/zalando/fahrschein/NakadiReader.java

```diff
@@ -200,13 +199,15 @@ private Cursor readCursor(JsonParser jsonParser) throws IOException {
         return events;
     }
 
-    private void readEvent(final Iterator<T> source, final List<T> target) throws JsonMappingException {
+    private void readEvent(final Iterator<T> source, final List<T> target) throws IOException {
         try {
             target.add(source.next());
```

Comment:
I'm wondering if the `hasNext` method is also allowed to throw runtime exceptions.